src/main/java/seedu/address/model/person/Person.java

```diff
@@ -120,14 +104,13 @@ public boolean equals(Object other) {
                 && otherPerson.getEmail().equals(getEmail())
                 && otherPerson.getDob().equals(getDob())
                 && otherPerson.getAddress().equals(getAddress())
-                && otherPerson.getTags().equals(getTags())
                 && otherPerson.getGender().equals(getGender()); // add gender field for comparison
     }
 
     @Override
     public int hashCode() {
         // use this method for custom fields hashing instead of implementing your own
-        return Objects.hash(name, phone, email, address, tags, gender); //add gender field
+        return Objects.hash(name, phone, email, address, gender); //add gender field
```

Comment:
Let's remove this comment since it's already present now
```suggestion
        return Objects.hash(name, phone, email, address, gender); 
```